adafruit_bitbangio.py

```diff
@@ -261,7 +269,7 @@ def _write(self, address, buffer, transmit_stop):
         if transmit_stop:
             self._stop()
 
-    def _read(self, address, length):
+    def _read(self, address: int, length: int) -> ReadableBuffer:
```

Comment:
Since `buffer` is set to a `bytearray` specificly, the return should also be as specific as possible, so it should return `byetarray`.